tests/suites/test_suite_psa_crypto_generate_key.generated.data

```diff
@@ -109,11 +109,11 @@ depends_on:PSA_WANT_KEY_TYPE_RAW_DATA
 generate_key:PSA_KEY_TYPE_RAW_DATA:128:PSA_SUCCESS:
 
 PSA RSA_KEY_PAIR 1024-bit
-depends_on:PSA_WANT_KEY_TYPE_RSA_KEY_PAIR:MBEDTLS_GENPRIME
+depends_on:PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_GENERATE:PSA_VENDOR_RSA_GENERATE_MIN_KEY_BITS <= 1024:DEPENDENCY_NOT_IMPLEMENTED_YET
```

Comment:
`PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_GENERATE` doesn't exist in 2.28. The old dependencies were correct for 2.28.